source/style-guide/style/punctuation.txt

```diff
@@ -4,12 +4,10 @@
 Punctuation
 ===========
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 Use punctuation correctly and consistently. This section provides
 guidelines for using punctuation in MongoDB documentation. For basic
 rules about punctuation, see a grammar reference, such as the
-*Harbrace College Handbook*.
+*Chicago Manual of Style*.
```

Comment:
**suggestion:** CMOS should be our official reference. 
```
see :cmos17:`section 5, Grammar and Usage </part2/ch05/toc.html>` in the *Chicago Manual of Style*.

Reply:
I agree, just missed it on this page. Thanks for 👀